lib/IRGen/IRGenSIL.cpp

```diff
@@ -1991,43 +2014,63 @@ void IRGenSILFunction::visitGraphOperationInst(GraphOperationInst *i) {
   auto *getContextFn = IGM.getTFC_GetGlobalEagerContextFn();
   auto eagerContext = Builder.CreateCall(getContextFn, {});
 
-  // Create a TFE op as in:
-  //   auto* op = TFE_NewOp(ctx, "OpName", status);
+  LLVM_DEBUG(llvm::dbgs() << "Calling TFE_NewOp(ctx, op, status) with op name "
+                          << opInfo.getOperationName() << ".\n");
   auto *TFENewOpFn = IGM.getTFE_NewOpFn();
   auto opNameValAddr = createStringValAddr(IGM, opInfo.getOperationName());
   auto op =
       Builder.CreateCall(TFENewOpFn, {eagerContext, opNameValAddr, status});
   checkOk(status);
 
+  auto addTensorhandleAsOpInput = [=](SILValue tensorHandle) {
```

Comment:
Hmm, why `[=]`? `IGM` and `Builder` shouldn't be copied. I'm surprised this didn't result into strange behavior or assertion.

Reply:
Changed to & to be safer. Perhaps copying them is fine -- If these objects cannot be copied, usually their copy c'tor should be deleted/disabled. But avoiding copying is better.

Reply:
Their copy c'tors should've been deleted. These objects are very, very large structs.